sources/api/schnauzer/src/v2/import/fake.rs

```diff
@@ -0,0 +1,81 @@
+//! Provides a helpful "fake" implementation of TemplateImporter to use in test scenarios.
+use async_trait::async_trait;
+use handlebars::Handlebars;
+
+use crate::v2::ExtensionRequirement;
+
+use super::{HelperResolver, SettingsResolver};
```

Comment:
Nit: We've typically been grouping all our imports together.  (`rust-analyzer` has a setting for this; it likes to split them up)